deployments/cloudfoundry/bosh/release

```diff
@@ -81,5 +81,6 @@ else
   # If we're doing an official release we want to delete dev releases, and create a tarball of the release
   # to be used by the tile.
   rm -rf $SCRIPT_DIR/.dev_builds $SCRIPT_DIR/dev_releases
-  bosh create-release --force  --version ${OTEL_VERSION}-${BOSH_RELEASE_VERSION} --tarball ./latest-release.tgz --name splunk-otel-collector
+  bosh create-release --force --version ${OTEL_VERSION}-${BOSH_RELEASE_VERSION} --tarball ./latest-release.tgz --name splunk-otel-collector
+  rm -rf $SCRIPT_DIR/splunk_otel_collector $SCRIPT_DIR/signalfx-agent.tar.gz
```

Comment:
Is `signalfx-agent.tar.gz` still required?  The collector package already includes the agent bundle.

Reply:
Good question, I'll go test without it.

Reply:
I don't believe it's required. The Splunk Collector provides the required functionality for monitoring the cloud foundry environments, and I'm not aware of any reason a customer would want to configure the collector to have the signalfx receiver enabled.

The bosh release and Tanzu Tile both work properly without the agent. I'll remove the smart agent references from both.